web/geo/detect-changes.js

```diff
@@ -0,0 +1,90 @@
+const topojsonClient = require("topojson-client");
+const {  getCoords, featureCollection, area, multiPolygon } = require("@turf/turf")
+const fs = require("fs");
+const { getPolygons } = require("./utilities");
+
+const getJSON = (fileName, encoding = "utf8", callBack = () => { }) =>
+    typeof fileName === "string" ?
+        JSON.parse(fs.readFileSync(fileName, encoding, () => callBack())) :
+        fileName;
```

Comment:
This function exists in utilities

Reply:
Just wanted to sneak in some extra code lines